src/Okta.Sdk/Internal/CollectionAsyncEnumerator{T}.cs

```diff
@@ -82,15 +82,15 @@ public sealed class CollectionAsyncEnumerator<T> : IAsyncEnumerator<T>
         private static HttpRequest GetNextLink(HttpResponse response)
         {
             var linkHeaders = response
-                .Headers
+                .Headers?
                 .Where(kvp => kvp.Key.Equals("Link", StringComparison.OrdinalIgnoreCase))
                 .Select(kvp => kvp.Value);
 
-            var nextUri = LinkHeaderParser
+            var nextUri = (linkHeaders != null) ? LinkHeaderParser
```

Comment:
Instead of a ternary/inline if, maybe split it up for readability:

```
var nextUri = string.Empty;
if (linkHeaders != null) {
  nextUri = LinkHeaderParser...
```